python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -29,26 +29,19 @@ def app_upgrade_info(engine):
 
     :param engine: The currently-running engine instance.
     """
-    import sgtk
-    logger = sgtk.LogManager.get_logger(LOGGER_NAME)
-
-    code_css_block = "display: block; padding: 0.5em 1em; border: 1px solid #bebab0; background: #faf8f0;"
-
-    logger.info(
+    engine.log_info(
```

Comment:
please add a comment that this uses markdown and this is safe because this code will only ever be executed on sg sites with 7.2+